decidim/decidim
# Fix background-image URLs with weird characters

#### :tophat: What? Why?

While publishing the next DecidimFest, we saw that images with weird characters in their filename (like `banner-web(1).png`) didn't get displayed correctly after uploading them. 

This was happening in the "Hero image" homepage Content Block, but its happening in other places as this is a pattern that we repeat in other views. 

This PR fixes that. 

#### Testing

1. Sign in as admin
2. Go to http://localhost:3000/admin/organization/homepage/content_blocks/hero/edit
3. Upload a background image with a filename like "banner-web(1).png"
4. Click on the "Update" button
5. See that the image isn't correctly shown 


### :camera: Screenshots

#### Before
![Selection_243](https://user-images.githubusercontent.com/717367/175295513-712fe2f1-e6d4-47a6-add8-16756b92edcc.png)


#### After 
![Selection_244](https://user-images.githubusercontent.com/717367/175295547-035d917c-6ca9-4fcd-8eb8-fd0b6eae0cb3.png)

:hearts: Thank you!
